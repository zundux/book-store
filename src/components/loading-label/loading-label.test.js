import React from "react";
import LoadingLabel from "./loading-label";

describe("Loading label", () => {

  test("Loading label has only the message \"Loading, please wait\" ", () => {
    const component = shallow(
      <LoadingLabel/>
    );
    matchSnapshot(component);
  });
});

