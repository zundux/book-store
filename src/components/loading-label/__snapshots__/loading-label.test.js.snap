// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading label Loading label has only the message "Loading, please wait"  1`] = `
<div>
  Loading, please wait
</div>
`;
